### PR TITLE
Updated index.md fixed sample which was failing.

### PR DESCRIPTION
**Pull request by naveensrinivasan**
Updated index.md fixed sample which was failing.

### CODE REVIEW

File: docs/user-guide/configmap/index.md [hunk 1]
```diff
@@ -362,7 +362,7 @@ spec:
   containers:
     - name: test-container
       image: gcr.io/google_containers/busybox
-      command: [ "/bin/sh", "cat", "/etc/config/special.how" ]
```
Comment (pwittrock) [inline docs/user-guide/configmap/index.md]: instead how about just adding "-c" as shown in the other commands that use /bin/sh.  Also would you update the next occurrence as well.

**Comment by pwittrock** [comment]
@naveensrinivasan Left one comment.

**Comment by pwittrock** [comment]
@naveensrinivasan is this still active?

**Comment by naveensrinivasan** [comment]
@pwittrock Was busy with other things. Will look at it and get back to you soon. Thanks

**Comment by pwittrock** [comment]
@naveensrinivasan Thanks.  This looks good.  Would you just resolve the merge conflicts?

**Comment by naveensrinivasan** [comment]
I don't have the patch branch, I used the github site to for the PR. I could close this one and then open another one for this patch. I am not able to resolve the conflict. Please let me know.

Thanks

**Comment by pwittrock** [comment]
try

```
git checkout -b naveensrinivasan-patch-1 master
git pull https://github.com/naveensrinivasan/kubernetes.github.io.git patch-1
```

**Comment by pwittrock** [comment]
I fixed the conflicts in https://github.com/kubernetes/kubernetes.github.io/pull/997 and kept you as the commit author.  Would you add a comment to that PR saying LGTM if it looks good to you and I will submit.

**Comment by pwittrock** [comment]
Merged in #997

**Comment by pwittrock** [comment]
@naveensrinivasan Thanks for your contribution

**Pull request merged**